prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1260,18 +1260,22 @@ struct fi_ops_cm rxr_ep_cm = {
 };
 
 /*
- * @brief explicitly allocate a chunk of memory for 5 packet pools on RX side:
+ * @brief explicitly allocate a chunk of memory for 6 pools on RX side:
  *     efa's receive packet pool (efa_rx_pkt_pool)
  *     shm's receive packet pool (shm_rx_pkt_pool)
  *     unexpected packet pool (rx_unexp_pkt_pool),
  *     out-of-order packet pool (rx_ooo_pkt_pool), and
  *     local read-copy packet pool (rx_readcopy_pkt_pool).
+ *     read entry pool (read_entry_pool)
+ *
+ * This function is called when the progress engine is called for
+ * the 1st time on this endpoint.
  *
  * @param ep[in]	endpoint
  * @return		On success, return 0
```

Comment:
Not a change you made but:
 - I think this should be `@param[in,out] ep`  rather than `@param ep[in]`

Reply:
Fixed